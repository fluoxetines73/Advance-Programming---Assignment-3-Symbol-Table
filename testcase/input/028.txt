RPRINT
BEGIN
ASSIGN e_123_2r unimportant
RPRINT
RPRINT
RPRINT
PRINT
END