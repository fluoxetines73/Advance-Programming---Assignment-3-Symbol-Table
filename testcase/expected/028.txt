Undeclared: ASSIGN e_123_2r unimportant